model/client4.go

```diff
@@ -2649,7 +2649,7 @@ func (c *Client4) UploadBrandImage(data []byte) (bool, *Response) {
 
 // GetLogs page of logs as a string array.
 func (c *Client4) GetLogs(page, perPage int) ([]string, *Response) {
-	query := fmt.Sprintf("?page=%v&per_page=%v", page, perPage)
+	query := fmt.Sprintf("?page=%v&logs_per_page=%v", page, perPage)
```

Comment:
Why the rename? The names used by the API are generally set in stone unless there's a very good reason to change them.

Reply:
I have asked about this in [issue comment](https://github.com/mattermost/mattermost-server/issues/6724#issuecomment-341873681) and @jwilander also agree this change.

Reply:
I see:

> I think it makes sense to add the new query parameter. I'd prefer going with logs_per_page to match the constant names

Just to make absolutely sure there was no confusion: This isn't a *new* query parameter, this is renaming a previously existing and documented parameter. If that's still okay with @jwilander, I'm fine with it considering the parameter didn't even work. But I want to make sure it's clear that this is in some ways a backwards-incompatible change.